mobify/platform-scaffold
# [IM to SDK] Move app branch

This PR removes the home and custom branches from the scaffold Integration manager. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-21
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/692

## Changes
- Removed app branch from the Integration Manager
- Updated paths for app commands to point to the SDK versions

## How to test-drive this PR
- checkout the im-app-branch branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the merlins site
- navigate to a PDP and add the item to your cart
- refresh the page
- the cart icon in the header should show the correct cart count (1) this means the initApp command ran and fetched the cart data
